src/constant.js

```diff
@@ -20,6 +20,7 @@ export const M = 'month'
 export const Q = 'quarter'
 export const Y = 'year'
 export const DATE = 'date'
+export const UNITS = [MS, S, MIN, H, D, W, M, Q, Y, DATE]
```

Comment:
all right, that makes sense.

The last thing we should take care of is how to make this constant file as small as possible. Since it will bundle to the main bundle file.

Maybe we can try to use the units above? (not sure)

Reply:
Good point. We're fetching object values for each of them explicitly, it makes sense to import each of them separately.